mflowgen/Tile_MemCore/custom-init/outputs/floorplan.tcl

```diff
@@ -11,11 +11,13 @@
 #-------------------------------------------------------------------------
 
 # Density target: width will be adjusted to meet this cell density
-set core_density_target 0.70; # Placement density of 70% is reasonable
+set core_density_target 0.55; # Placement density of 70% is reasonable
 # Core height : number of vertical pitches in height of core area
 # We fix this value because the height of the memory and PE tiles
 # must be the same to allow for abutment at the top level
-set core_height 139
+set core_height 150
```

Comment:
Can you make the accompanying change in the Tile_PE floorplanning script? It has to be the same height.

Reply:
resolved in latest commit